profile/README.md

```diff
@@ -118,14 +118,15 @@ OpenEBS is the most successful Stateful Persistent Container Native Storage plat
 ## Community Resources
 We centralize all our community documentation in the [Community](https://github.com/openebs/community) repo. All sub-projects within the organization follow the same umbrella policies, except where stated in the sub-project readme.
 
-## Monthly community meeting
-OpenEBS holds a monthly community meeting via Zoom on the last Thursday of the month, at 7 am Pacific time.
+## Monthly Community Meetings
```

Comment:
There's a link on the top of the page to this section. Can you check if that needs an update as well?

Reply:
Done.